zulipterminal/config/themes.py

```diff
@@ -1,58 +1,9 @@
 from typing import Dict, List, Optional, Tuple
 
+from zulipterminal.themes._template import REQUIRED_STYLES
```

Comment:
I wanted to make template look like the other themefiles so that we can direct people to template to understand the format.
Having default colors in template make more sense as it tells how a theme like gruvbox should be made, else it would look like you would have to import color from a color file.

Reply:
Having the example theme files should give an idea of how to do it, and gruvbox is an example of separate colors. Having a FAQ entry or a separate short theme writing doc would work fine to describe this.

I'm not saying a template file is a bad idea, just that default colors/styles belong somewhere else.

If someone just edits the template file they'll change default colors/styles and cause problems.

Reply:
It if fine if they edit default color right? Maybe they want to change 1 or 2 colors. Even Gruvbox colors can be edited the same way.

Reply:
My expectation would be that editing a 'template' theme file would only affect the new theme that I'm writing based on the template.

That's not to say that changing some base colors should be difficult, just that this feels like it would be confusing.

Reply:
Hmm...would a name change to `defaults`, clear the confusion? I could move all the doc-strings to a MD file. `defaults` would contain default colors and default mono theme ( technically its the default mono theme for all themes and not Required Styles ).

Reply:
Re script, a simple copy paste of an existing theme would be enough? A script here is too much I think.

Reply:
Having Color in `themes.py` would cause a circular import problem.
themes,py -> zt_dark.py -> themes.py

Reply:
I have made a Common's file in themes where I have placed the default color scheme and required styles.
I have also separated the documentation into a THEME_CONTRIBUTING.md file, not sure where it should go so its currently in the themes folder.
I'm thinking lets put it in the same directory level as README